src/views/start.html

```diff
@@ -1,11 +1,31 @@
 {% extends "layout.html" %}
 
 {% block content %}
-    <div class="govuk-grid-row">
+
+<div class="govuk-width-container">
+    <main class="govuk-main-wrapper">
+      <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-            <h1 class="govuk-heading-xl">GitHub Requests</h1>
+            <h1 class="govuk-heading-l">GitHub Requests</h1>
+
+            <p class="govuk-body">The GitHub request application is a tool designed to streamline and automate the process of managing and tracking requests for GitHub. This includes the adding, removal or editing of members. Further documentation can be found <a href="https://github.com/cabinetoffice/github-requests-app" class="govuk-link">here</a>.</p>
+
+            <h1 class="govuk-heading-m">Before you start:</h1>
 
-            <p class="govuk-body-l">Use this service to make member, team, and repository related requests to the cabinetoffice GitHub organisation.</p>
+            <ul class="govuk-list govuk-list--bullet govuk-list--spaced">
+                <li>
+                    <p class="govuk-body">New users must have a GitHub handle.</p>
+                </li>
+                <li>
+                    <p class="govuk-body">New user has to be associated with an existing team; otherwise, the new team must be created in the same request.</p>
```

Comment:
"New users have" as consistent with first point... 

Now that we can add outside collaborators, I think we need a caveat saying that they are exempt from this rule?